apps/condo/domains/acquiring/schema/Payment.js

```diff
@@ -92,27 +106,27 @@ const Payment = new GQLListSchema('Payment', {
 
         advancedAt: {
             schemaDoc: 'Time at which money was advanced to recipient\'s account',
-            type: DateTimeUtc,
+            type: 'DateTimeUtc',
```

Comment:
Why you change a type from wrapper to a keystone one?

Reply:
All new models use string values of types and mapping during schema registration. As I remember, it's needed to help us to migrate to ks6 in the future. In the far future. In the very far future :)